README.md

```diff
@@ -26,6 +26,20 @@ The user **must** provide a list of priority classes names when deploying the
 policy. Both `allowed_priority_classes` and `denied_priority_classes` cannot
 be used at the same time.
 
+If the `PodSpec` of the resource does not have the `priorityClassName` field
+defined, it's possible to configure the policy to mutate the resource adding a
+default value. This is possible by defining the `default_priority_class`
+settings. This field is optional. But when defined, it should be listed in the
+`allowed_priority_classes` and not present in the `denied_priority_classes`.
+When the resource has a `priorityClassName` defined, no mutation is performed.
+And the validations against the settings provided are done as usual.
+
+> [!WARNING]
+> It's important that the `PriorityClass` name defined in the
+> `denied_priority_classes` setting actually exists in the cluster. Otherwise,
+> the policy will mutate resources to use a non-existent class, which will
+> cause Kubernetes to reject them.
```

Comment:
@kravciak what do you think?

Reply:
I like it, but I think it has wrong setting :)

Reply:
Argh, autocomplete is sabotaging... xD